randovania/cli/commands/export_db_videos.py

```diff
@@ -283,6 +286,59 @@ def filename_friendly_game_name(game: RandovaniaGame):
     return "".join(x for x in game.long_name if x.isalnum() or x in [" "])
 
 
+def export_as_yaml(game: RandovaniaGame, out_dir: Path, as_frontmatter: bool):
```

Comment:
Not gonna point out every single one of them: are all the mypy errors intended?

Reply:
yes, insofar as virtually every other CLI function would have the same errors and it seems out of scope to fix them all